src/GrpAb/Dual.jl

```diff
@@ -5,7 +5,7 @@
 #
 ######################################################################
 
-struct QmodnZ <: GrpAb
+mutable struct QmodnZ <: GrpAb
```

Comment:
I had to make this `struct` mutable because the `WeakValueDict` wants to add a finalizer.

Reply:
OK